.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -e .
-          pip install pytest==6.1.1 torchvision pytorch_lightning==1.1.1 scikit-learn gorilla transformers torchtext matplotlib captum
+          pip install pytest==6.1.1 torchvision pytorch_lightning==1.2.0 scikit-learn gorilla transformers torchtext matplotlib captum
```

Comment:
@Arvind-Ideas2IT Please change it to `pytorch_lightning>=1.2.0`. So that we will get to know if anything goes wrong in the newer versions.

Reply:
@shrinath-suresh  Done. Checked all the examples with PTL 1.2.1 and working seamlessly